backend/src/resolvers/Mutation/Concept.js

```diff
@@ -104,9 +105,14 @@ export const createConcept = async (root, {
     position,
     official: Boolean(official),
     frozen: Boolean(frozen),
-    course: { connect: { id: courseId } },
     tags: tags && { connect: await createMissingTags(tags, workspaceId, context, 'conceptTags') }
-  })
+  }
+
+  if (Boolean(courseId)) {
+    conceptData.course = { connect: { id: courseId } }
```

Comment:
Omitting the course ID should only be allowed to goals and goals should only be allowed for staff